index.html

```diff
@@ -0,0 +1,153 @@
+          /></a>
+        </li>
+        <li class="home"><a href="./index.html">Home</a></li>
+        <li><a href="./index.html#organizers">Organizers</a></li>
+        <li><a href="./index.html#program">Program</a></li>
+        <li><a href="./index.html#speakers">Speakers</a></li>
+        <li class="about"><a href="./about.html">About</a></li>
+      </ul>
+    </nav>
+
+    <header>
+      <h2>"Hello! Sharing world"</h2>
+      <h1>EASY SEND INTERNATIONAL CONFERENCE</h1>
+      <div>
+        <p>
+          A joyful celebration believing the value of openness and sharing, a
+          positive change with people from over 80 countries is taking place in
+          october, in Jos, Plateau State Nigeria
+        </p>
+        <h3>2022.04.01(FRI) ~ 02(SAT)</h3>
+        <h4>&commat; EASSEND Conference center Millioniar's Quaters Jos</h4>
+      </div>
+    </header>
+
+    <section class="program" id="program">
+      <h2>Main Program</h2>
+      <hr />
+      <div class="program-cards">
+        <div class="card">
+          <img src="./images/icon_05.png" alt="lecture" />
+          <h3>Lecture</h3>
+          <p>
+            Listen to the speakers from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/icon_02.png" alt="exhibition" />
+          <h3>Greeting</h3>
+          <p>
+            Appreciate various creations applying CC license of artists,
+            organized from Art Center Nabi
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/icon_03.png" alt="Forum" />
+          <h3>Forum</h3>
+          <p>
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/icon_04.png" alt="Workshop" />
+          <h3>Technology Interchange</h3>
+          <p>
+            Try creating your own work using open source license rather than
+            just watching at it.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/icon_01.png" alt="cc ignite" />
+          <h3>Alien Speakers</h3>
+          <p>
+            Get opportunities to network with CC affiliates around the world,
+            also after the summit.
+          </p>
+        </div>
+      </div>
+      <div class="btn-container">
+        <button type="button" class="join-conference-btn">
+          Join EASYSEND SUMMIT 2022
+        </button>
+      </div>
+      <div class="see-program">SEE MORE</div>
+    </section>
+
+    <section class="speakers" id="speakers">
+      <h2>Featured Speakers</h2>
+      <hr />
+      <div class="speakers-cards">
+        <div class="featured-speakers grid"></div>
+        <button type="button" id="more">MORE<span>v</span></button>
+        <div class="important-speakers grid"></div>
+      </div>
```

Comment:
As requested by the previous code reviewer,  kindly include the second light image behind the main image.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/159101730-915522d8-9fef-4f18-92d2-cd46e93e30a3.png) | ![image](https://user-images.githubusercontent.com/10332499/159101751-4ae417f4-3b4e-4a9e-b238-29f2f18f44a6.png)